pkg/virt-controller/watch/workload-updater/workload-updater.go

```diff
@@ -289,11 +289,20 @@ func (c *WorkloadUpdateController) getUpdateData(kv *virtv1.KubeVirt) *updateDat
 		lookup[migration.Namespace+"/"+migration.Spec.VMIName] = true
 	}
 
+	liveMigrationFGEnabled := false
```

Comment:
You can do `c.clusterConfig.isFeatureGateEnabled(virtconfig.LiveMigrationGate)`

Reply:
you're right! I missed the controller had the clusterConfig stored, thanks!